src/Beacon/BeaconDemo/BeaconDemo.csproj

```diff
@@ -1,113 +1,18 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{D0B707E4-5563-41FA-A8B0-292605332EF1}</ProjectGuid>
+    <TargetFrameworks>net45;net47;net5.0;netcoreapp3.1;netcoreapp3.1</TargetFrameworks>
+    <AssemblyVersion>1.0.0.0</AssemblyVersion>
+    <FileVersion>1.0.0.0</FileVersion>
```

Comment:
I never used for PackageReference, but if it works I agree, I will look into.

Reply:
It will work for `PackageReference`. You can think of `Directory.Build.props` as containing regular project file elements, that is automatically inserted at the top of your project file. In that way, it can specify defaults, and a given project may override those defaults if needed.

Reply:
done